### PR TITLE
Add license and code of conduct

### PR DESCRIPTION
**Pull request by worldtiki**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (fabianpiau) [inline LICENSE:189]: Are you not supposed to replace the placeholder here?

Comment (massdosage) [reply LICENSE:189]: This is just boilerplate that GH itself generates when you select a license type when creating a new project. I've always viewed it as a copy of the original Apache license to identify what the project uses, and then each source code file adds whatever copyright it wants. Most projects I've seen on GH leave this "as is" but some do fill it in. I personally prefer just leaving it, one less thing to maintain...

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behaviour may be
+reported by contacting [a member of the project team](https://github.com/orgs/ExpediaGroup/teams/helm-charts-committers/members). All
```
Comment (nikos912000) [inline CODE-OF-CONDUCT.md:58]: Have we created this team? Doesn't seem to exist.

Comment (massdosage) [reply CODE-OF-CONDUCT.md:58]: Don't think so, I can, who should be in it and who wants to be admin for the team?

Comment (worldtiki) [reply CODE-OF-CONDUCT.md:58]: odd. i created it. maybe i didnt make it public :|

Comment (nikos912000) [reply CODE-OF-CONDUCT.md:58]: Thanks Adrian. You can add @worldtiki and myself as admins if that's ok.
Can add more people later.

Comment (worldtiki) [reply CODE-OF-CONDUCT.md:58]: oops. typo. 1 t in committer :)

Comment (nikos912000) [reply CODE-OF-CONDUCT.md:58]: Ah hold on, it's [here](https://github.com/orgs/ExpediaGroup/teams/helm-charts-commiters). Note that it's `commiters` not `committers`, can we fix the typo in the team name?

Comment (worldtiki) [reply CODE-OF-CONDUCT.md:58]: yes, it's fixed. the link in the document works now

**Pull request merged**